codespell_lib/data/dictionary.txt

```diff
@@ -5735,6 +5737,7 @@ diposed->disposed, deposed,
 diposing->disposing, deposing,
 dipthong->diphthong
 dipthongs->diphthongs
+dircet->dircet
```

Comment:
This is a self-replacement, I think the correction should be direct.

Reply:
Done